silkworm/core/execution/call_tracer.cpp

```diff
@@ -18,11 +18,58 @@
 
 #include <evmc/hex.hpp>
 #include <evmc/instructions.h>
+#include <evmone/baseline_instruction_table.hpp>
 #include <evmone/execution_state.hpp>
 #include <evmone/instructions.hpp>
 
 #include <silkworm/core/types/address.hpp>
 
+using namespace evmone;
+using namespace evmone::baseline;
+
+// The following check_requirements function has been temporarily copied from evmone because it is not exported.
+// We need to ask evmone for it to be exported or for a tracing interface extension (e.g. on_instruction_end?).
```

Comment:
My vote is for the tracing interface extension, seems to be more aligned with the current code